docs/ja/mkdocs.yml

```diff
@@ -17,14 +18,24 @@ edit_uri: ''
 google_analytics:
 - UA-133183413-1
 - auto
+plugins:
+- search
+- markdownextradata:
+    data: data
 nav:
 - FastAPI: index.md
 - Languages:
   - en: /
   - es: /es/
+  - fr: /fr/
+  - it: /it/
   - ja: /ja/
   - pt: /pt/
+  - ru: /ru/
+  - tr: /tr/
+  - uk: /uk/
   - zh: /zh/
+- alternatives.md
```

Comment:
Only this line is edited manually, others are changed automatically by `python scripts/docs.py build-all
`.